src/Control/Distributed/Process/Tests/CH.hs

```diff
@@ -734,7 +745,8 @@ testReconnect TestTransport{..} = do
 
 
     -- Simulate network failure
-    liftIO $ testBreakConnection (nodeAddress nid1) (nodeAddress nid2)
+    liftIO $ do testBreakConnection (nodeAddress nid1) (nodeAddress nid2)
+                threadDelay 1000000
```

Comment:
`threadDelay` is so evil. Because it doesn't guarantee the absence of races. And slows tests down. Can we avoid this?